laws/src/main/scala/cats/laws/discipline/MiniInt.scala

```diff
@@ -0,0 +1,36 @@
+package cats
+package laws
+package discipline
+
+import cats.instances.int._
+
+/**
+ * Similar to `Int`, but with a much smaller domain. The exact range of [[MiniInt]] may be tuned
+ * from time to time, so consumers of this type should avoid depending on its exact range.
+ */
+final class MiniInt private (val asInt: Int) extends AnyVal with Serializable
```

Comment:
Why not just use Short or Byte or something?

Reply:
Do you mean on the inside of the `MiniInt` or using `Byte` or `Short` directly without a wrapper?

I chose to use a custom type so that we could tune the range as desired. A lot of times these checks come up when a law is checking the equality inside of a `forAll`, so we'll end up with 50 items in an outer loop (based on our Scalacheck config). If we were to use byte then we'd end up with 50 * 256 comparisons (or perhaps 50 * 256 * 256 if we are comparing two `Function2` instances for equality). So I thought that something with a smaller domain might be desirable.

If you were just referring to why not use `Byte` instead of `Int` inside the `MiniInt` class, then I have no good reason and can change if you'd like.

In case it's of interest at one point I went down a rabbit hole of reimplementing parts of [smallcheck](http://hackage.haskell.org/package/smallcheck) within Cats laws so that we could have tests declare the "depth" that they wanted to use when comparing for equality. But it seemed like overkill and testing over a subset of a domain of `Int` didn't feel as safe to me as testing against the entire domain of a smaller type.